packages/shared/src/components/notifications/InAppNotification.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { useQuery, useQueryClient } from 'react-query';
+import classNames from 'classnames';
+import {
+  IInAppNotification,
+  IN_APP_NOTIFICATION_KEY,
+} from '../../hooks/useInAppNotification';
+import classed from '../../lib/classed';
+import { isTouchDevice } from '../../lib/tooltip';
+import CloseButton from '../CloseButton';
+import { InAppNotificationItem } from './InAppNotificationItem';
+import styles from './InAppNotification.module.css';
+
+const Container = classed(
+  'div',
+  classNames(
+    styles.inAppNotificationContainer,
+    'animate-bounce',
+    'fixed bottom-10 right-10 bg-theme-bg-secondary border border-theme-active rounded-16 in-app-notification slide-in z-[10] w-[22.5rem] h-22',
+  ),
+);
+
+let timeoutId: number | NodeJS.Timeout = 0;
+
+export function InAppNotification(): ReactElement {
+  const router = useRouter();
+  const client = useQueryClient();
+  const [isExit, setExit] = useState(false);
+  const closeNotification = () => {
+    setExit(true);
+    setTimeout(() => {
+      setExit(false);
+      client.setQueryData(IN_APP_NOTIFICATION_KEY, null);
+    }, 150);
+  };
+  const stopTimer = () => clearTimeout(timeoutId);
+  const startTimer = (timer: number) => {
+    timeoutId = setTimeout(closeNotification, timer);
```

Comment:
It's a good practice to clear the timeout before we set anyone just in case it was set for some odd reason

Reply:
Done 👍